values-schema.yaml

```diff
@@ -346,6 +444,12 @@ definitions:
     required:
       - name
       - type
+  shell:
+    default: sh
+    enum:
+      - sh
+      - bash
```

Comment:
why is this needed? it is encapsulated by the running script. otomi-core should not know about this.

Reply:
Right now it's encapsulated explicitly: `command: ["{{ $spec.init.shell | default "sh" }}", "-c"]`. Do you suggest to remove this reference?

Reply:
We don't want to know about what to run or how. Expect the tooling to be in the container and just use the default shell. No choosing here

Reply:
if they want to start another type of shell they can if they built it so